template.tpl

```diff
@@ -1409,8 +1408,45 @@ scenarios:
     // Verify that the tag finished successfully.
     assertApi('makeTableMap').wasCalledWith(mockData.advancedMatchingParams, 'name', 'value');
     assertApi('gtmOnSuccess').wasCalled();
+- name: Test Track Advanced Matching Parameters
```

Comment:
Could we add a test specifically for this flow:
- email/other being empty during init
- but then being set later during track

Reply:
So this test was somewhat doing that, but I made it more explicit.  I also added a second test for the values being set once, and then updated in the second GTM invocation.

Reply:
Perfect, thanks!